tests/unit/serialization/format_test.cpp

```diff
@@ -414,3 +415,54 @@ TEST_F(format_test, malformed_unordered_sets_cannot_roundtrip) {
     // member)
     EXPECT_FALSE(deser);
 }
+
+TEST_F(format_test, get_variant_default_constructibles) {
+    // empty set
+    std::unordered_set<uint64_t> s0;
+    std::variant<std::unordered_set<uint64_t>, uint64_t> v0{s0};
+    // how ser and deser are connected?
```

Comment:
This comment seems like it might be left over from implementation (doesn't look to be documenting anything).

Reply:
removed